pom.xml

```diff
@@ -66,45 +66,55 @@
 
     <properties>
         <geomesa.version>4.1.0-SNAPSHOT</geomesa.version>
-        <nifi.version>1.23.0</nifi.version>
+        <nifi.version>1.23.2</nifi.version>
 
         <scala.version>2.12.17</scala.version>
         <scala.binary.version>2.12</scala.binary.version>
 
-        <avro.version>1.11.1</avro.version> <!-- version bundled with nifi 1.19.1 -->
-        <guava.version>31.1-jre</guava.version> <!-- version bundled with nifi 1.19.1 -->
+        <avro.version>1.11.2</avro.version>
+        <guava.version>32.1.2-jre</guava.version>
 
         <!-- versions from geomesa -->
         <gt.version>28.2</gt.version>
         <accumulo.version>2.0.1</accumulo.version>
         <thrift.version>0.12.0</thrift.version>
         <zookeeper.version>3.5.10</zookeeper.version>
-        <hadoop.version>3.3.4</hadoop.version>
-        <hbase.version>2.5.3-hadoop3</hbase.version>
+        <hadoop.version>3.3.6</hadoop.version>
+        <hbase.version>2.5.5-hadoop3</hbase.version>
         <hbase-1.version>1.4.14</hbase-1.version>
         <kafka.version>2.8.2</kafka.version>
         <zkclient.version>0.11</zkclient.version>
-        <postgres.version>42.5.1</postgres.version>
+        <postgres.version>42.6.0</postgres.version>
 
         <curator.version>4.3.0</curator.version>
-        <netty.version>4.1.85.Final</netty.version>
-        <fasterxml.jackson.version>2.14.1</fasterxml.jackson.version>
+        <netty.version>4.1.97.Final</netty.version>
+        <fasterxml.jackson.version>2.15.2</fasterxml.jackson.version>
         <fasterxml.woodstox.version>5.4.0</fasterxml.woodstox.version> <!-- managed from 5.3.0 -->
         <commons.net.version>3.9.0</commons.net.version> <!-- managed from 3.1 -->
         <commons.text.version>1.10.0</commons.text.version>
         <wildfly.openssl.version>1.1.3.Final</wildfly.openssl.version> <!-- managed from 1.0.7.Final -->
-        <gson.version>2.10</gson.version>
+        <gson.version>2.10.1</gson.version>
+        <snappy-java.version>1.1.10.3</snappy-java.version> <!-- managed from 1.1.1.6 -->
+        <protobuf-java.version>3.24.3</protobuf-java.version>
+        <protobuf-java.hbase1.version>2.6.1</protobuf-java.hbase1.version> <!-- needs to be 2.x for hbase1 -->
```

Comment:
is this also getting used for hbase 2? i couldn't quite tell... if so, we could rename to just `protobuf-java.hbase.version`

Reply:
It was using the same version (I wasn't sure either, actually), so I made this change.